### PR TITLE
feat: [FFM-5218]: Updating node onboarding example to one click

### PR DESCRIPTION
**Pull request by laura-mannus**
### Summary

Updating our Node.js example so it aligns with the new one click onboarding instructions

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by crystal-fong — APPROVED** [review]

File: src/modules/75-cf/strings/strings.en.yaml [hunk 1]
```diff
@@ -1412,11 +1413,10 @@ onboarding:
...
         try {
-          // Log the state of the flag every 10 seconds
-          setInterval(async () => {
-            const value = await client.boolVariation(flagName, target, false);
-            console.log('Flag variation:', value);
-          }, 10000);
+        // Event happens when connection to server is established
+            client.on(Event.READY, async (flagIdentifier) => {
+              console.log(`${flagIdentifier} ready`);
+            });
```
Comment (knagurski) [inline src/modules/75-cf/strings/strings.en.yaml:1419]: Looks like there's some indentation issues here
```suggestion
          // Event happens when connection to server is established
          client.on(Event.READY, async (flagIdentifier) => {
            console.log(`${flagIdentifier} ready`);
          });
```

File: src/modules/75-cf/strings/strings.en.yaml [hunk 2]
```diff
@@ -1377,12 +1380,10 @@ onboarding:
...
 
-      When you run the following sample, it will:
-        1. Connect to the Feature Flag service
-        2. Report the value of the {{flagName}} flag every 10 seconds until the connection is closed.
-        3. Listen for when the flag is enabled or disabled and return the new value
+      Create a js file called 'example.js'. Copy and paste the following complete code example into that file. 
+      You can use this to test the harnessappdemodarkmode Flag you created on the Harness Platform. 
 
       ```
       const { Client, Event } = require('@harnessio/ff-nodejs-server-sdk');
```
Comment (knagurski) [inline src/modules/75-cf/strings/strings.en.yaml:1389]: Actually, let's simplify the heck outa this and just use promises.

```node
const { Client } = require('@harnessio/ff-nodejs-server-sdk');

const apiKey = '{{apiKey}}';
const flagName = 'harnessappdemodarkmode';

const client = new Client(apiKey);

const target = {
  identifier: 'nodeserversdk',
  name: 'NodeServerSDK',
};

client.waitForInitialization()
  .then(() => client.boolVariation(flagName, target))
  .then(value => console.log(`${flagName} is ${value}`))
  .finally(() => process.exit(0))
```

File: src/modules/75-cf/strings/strings.en.yaml [hunk 3]
```diff
@@ -1412,11 +1413,10 @@ onboarding:
         await client.waitForInitialization();
 
         try {
-          // Log the state of the flag every 10 seconds
-          setInterval(async () => {
-            const value = await client.boolVariation(flagName, target, false);
-            console.log('Flag variation:', value);
-          }, 10000);
+        // Event happens when connection to server is established
+            client.on(Event.READY, async (flagIdentifier) => {
```
Comment (knagurski) [inline src/modules/75-cf/strings/strings.en.yaml:1417]: Just FYI, this event doesn't pass a flag identifier.

**Review by knagurski — COMMENTED** [review]
Let's simplify 😊

File: src/modules/75-cf/strings/strings.en.yaml (hunk 1 above)
Comment (laura-mannus) [reply src/modules/75-cf/strings/strings.en.yaml]: Thanks have fixed :)

File: src/modules/75-cf/strings/strings.en.yaml (hunk 2 above)
Comment (laura-mannus) [reply src/modules/75-cf/strings/strings.en.yaml]: Cheers - have simplified

File: src/modules/75-cf/strings/strings.en.yaml (hunk 3 above)
Comment (laura-mannus) [reply src/modules/75-cf/strings/strings.en.yaml]: Have changed to `flags` and removed `async` from the events

**Review by knagurski — APPROVED** [review]
Nice and simple 🌟

**Pull request merged**